src/components/FeedBanner/FeedBanner.js

```diff
@@ -31,7 +33,7 @@ export default function FeedBanner ({
   } else if (!group) {
     return null
   } else {
-    ({ bannerUrl, avatarUrl, name, location } = group)
+    ({ bannerUrl, avatarUrl, name, groupLocation } = group)
```

Comment:
this seems off to me, i know groups have a location, its not called groupLocation, does this work still?

Reply:
mmm super good point